content/study-configuration/destination.md

```diff
@@ -4,28 +4,48 @@ date: 2023-04-05T08:41:03+02:00
 draft: false
 ---
 
-Every study needs a destination, where do the recruitment ads send the users? Destinations need to be connected to Virtual Lab so that, not only does it know where to send people who click on the recruitment ads, but also so that it knows how to collect the information about those who become study participants and knows how to optimize ads.
+Every study needs a destination, where do the recruitment ads send the users to? 
 
-Towards that end, Virtual Lab supports a set a "destinations" and is written in such a way that it is easy to add a new destination. When configuring your study, you can create one or more destinations and each destination needs a unique name (key) that you can refer to it by.
+Destinations need to be connected to Virtual Lab so that, not only does it know where to send people who click on the recruitment ads, but also so that it knows how to collect the information about those who become study participants and knows how to optimize ads.
```

Comment:
Techically, we think of the `data source`, where one collects information, as separate from the destination. 

We're having more and more instances of why that actually makes sense. 

For example, right now we have a couple new projects where they want us to screen people in the chatbot then send them to an external web survey. One can imagine that if we integrated their websurvey platform, we would have a destination being Fly Messenger, but we would have the data source being the external web survey platform.